react/typings/events.d.ts

```diff
@@ -14,7 +15,6 @@ export interface PixelMessage extends MessageEvent {
     | SearchPageInfoData
     | UserData
     | CartIdData
-    | CartData
```

Comment:
The _apparently_ harmless culprit... 🫠

<img width="479" alt="CleanShot 2023-04-25 at 16 32 45@2x" src="https://user-images.githubusercontent.com/381395/234383504-ad76b220-37e8-4681-b85a-cb5a3c01652a.png">